OKTA-480004: update readme with info about screenshots

hossammourad-okta opened the pull request:
https://oktainc.atlassian.net/browse/OKTA-480004
Adding documentation about workflows screenshots.

Changes are welcome if you think something should be phrased differently.

shauntarves-okta commented on README.md:
```diff
@@ -141,6 +141,46 @@ Example: The shape a `details` object for a template with no flos and no tags
+### Flows screenshots
+For each workflow that contains flows, a screenshot of each flow should be added to the `${WORKFLOW_FOLDER}/resources` directory (create the `resources` directory if it doesn't exist). The purpose of this screenshot is to showcase your flow to users in Okta's catalog platform.
+
+The screenshot should cover the full flow, not only parts of it, also it should be in high resolution.
```
"...should cover the full flow, plus the beginning and ending cards" or similar to indicate we want to capture the "placeholder" cards

hossammourad-okta commented:
@shauntarves-okta I added the tutorial to the repo as a readme file because Confluence pages are private and having the tutorial over there is worthless. I added the tutorial readme link to the readme of the repo as well in the top.

shauntarves-okta reviewed (approved):
This tutorial is a really, really great addition. Good catch on moving it into this repo vice having it in the wiki!
